src/Apps/Artist/Routes/Overview/index.tsx

```diff
@@ -52,8 +55,28 @@ class OverviewRoute extends React.Component<OverviewRouteProps, State> {
     return showGenes
   }
 
+  trackingCollectionsRailTest() {
```

Comment:
I don't see the `trackingCollectionsRailTest` function invoked elsewhere in this file. If we want the `ExperimentViewed` event recorded, I think that we'll want to bring in a `TrackingProp` and and use `tracking.trackEvent()`.

I removed similar code after the Artist Insights A/B test ended: https://github.com/artsy/reaction/pull/1919/files

Reply:
This is just a placeholder until we are ready to implement the test, but maybe it is better to comment out for now, or to call it and comment that line out.

Reply:
FWIW, I'm pretty sure using the tracking prop is an anti-pattern in reaction and the guideline is to use a decorator. My work with this is still WIP, but I got this feedback over on #2030.

Reply:
yup, btw we should use the `@track` decorator for analytics events instead of calling `tracking.trackEvent()` directly whenever possible

Reply:
sounds good -- based this on the [artist test ](https://github.com/artsy/reaction/pull/1919/files#diff-bfc57d8ce9d60a71675e59322d85c6f3L68) but can def edit to use the decorator pattern.